src/audio/volume/volume.c

```diff
@@ -175,17 +175,19 @@ static struct comp_dev *volume_new(struct sof_ipc_comp *comp)
 		if (vol->min_value < VOL_MIN) {
```

Comment:
Use imperative mood in the subject line + fix typo:

eg: trace: Add pipeline id to volume traces

see: https://chris.beams.io/posts/git-commit/#imperative

Also, commit body should contain a short explanation of why this commit is needed